packages/contracts-bedrock/contracts/test/Encoding.t.sol

```diff
@@ -6,8 +6,27 @@ import { Encoding } from "../libraries/Encoding.sol";
 
 contract Encoding_Test is CommonTest {
     function test_nonceVersioning(uint240 _nonce, uint16 _version) external {
-        uint256 nonce = Encoding.addVersionToNonce(uint256(_nonce), _version);
-        uint16 version = Encoding.getVersionFromNonce(nonce);
+        uint256 nonce = Encoding.encodeVersionedNonce(uint256(_nonce), _version);
```

Comment:
I don't love that `encodeVersionedNonce` takes a `uint256` because it can only safely work up to `uint240` - I'd rather be explicit with that with its interface

Reply:
That's a really good point.